packages/react-priority-overflow/src/useIsOverflowGroupVisible.ts

```diff
@@ -0,0 +1,9 @@
+import { useOverflowContext } from './overflowContext';
+
+/**
+ * @param id - unique identifier for a group of overflow items
+ * @returns visibility state of the group
+ */
+export function useIsOverflowGroupVisible(id: string | number) {
+  return useOverflowContext(ctx => ctx.groupVisibility[id + '']);
```

Comment:
```suggestion
  return useOverflowContext(ctx => ctx.groupVisibility[id]);
```

Is this required? `packages/react-priority-overflow/src/useIsOverflowItemVisible.ts` does not have it.

Reply:
yeah the ids madness was during the prototype phase, I should go and fix all of that casting

Reply:
Done in [edf7809](https://github.com/microsoft/fluentui/pull/21979/commits/edf780949ea82bacbfa43ab9a41d6809c390cd8b)